apps/mobile-wallet/src/navigation/RootStackNavigation.tsx

```diff
@@ -38,15 +38,20 @@ import ReceiveNavigation from '~/navigation/ReceiveNavigation'
 import RootStackParamList from '~/navigation/rootStackRoutes'
 import SendNavigation from '~/navigation/SendNavigation'
 import { loadBiometricsSettings } from '~/persistent-storage/settings'
-import { getDeprecatedStoredWallet, getStoredWallet, migrateDeprecatedMnemonic } from '~/persistent-storage/wallet'
+import {
+  getDeprecatedStoredWallet,
+  getStoredWallet,
+  migrateDeprecatedMnemonic,
+  storedWalletExists
+} from '~/persistent-storage/wallet'
 import AddressDiscoveryScreen from '~/screens/AddressDiscoveryScreen'
 import EditAddressScreen from '~/screens/Addresses/Address/EditAddressScreen'
 import NewAddressScreen from '~/screens/Addresses/Address/NewAddressScreen'
 import ContactScreen from '~/screens/Addresses/Contact/ContactScreen'
 import EditContactScreen from '~/screens/Addresses/Contact/EditContactScreen'
 import NewContactScreen from '~/screens/Addresses/Contact/NewContactScreen'
 import CustomNetworkScreen from '~/screens/CustomNetworkScreen'
-import LandingScreen from '~/screens/LandingScreen'
+import LandingScreen, { CoolAlephiumCanvas } from '~/screens/LandingScreen'
```

Comment:
You'l probably disagree with the name but whatever :P

Reply:
Mouhaha, I like it ;)